src/include/dhcp-server/widgets.rb

```diff
@@ -1319,5 +1321,11 @@ def service_widget_content
         }
       end
     end
+
+  private
+
+    def firewalld
```

Comment:
np: it was not needed as commented in #37 . It is defined by the interfaces helper https://github.com/yast/yast-yast2/blob/master/library/network/src/lib/y2firewall/helpers/interfaces.rb#L39